plugins/orchestrator-backend/dist-dynamic/package.json

```diff
@@ -53,6 +53,7 @@
   "dependencies": {
     "@octokit/rest": "^19.0.3",
     "@severlessworkflow/sdk-typescript": "^3.0.3",
+    "@urql/core": "^4.1.4",
```

Comment:
This is one example where cherry-picks won't be as easy as we wish.
`@urql/core` was introduced in another PR that is not on `KOGITO-9778` because likely it will be removed.
Although having this dependency there is not harmful, I believe we should keep our "cherry-picks" as clean as possible, i.e. adding only the absolutely necessary code related to the purpose of the PR.
WDYT?

Reply:
sure, removed that dependency from this PR.